listenbrainz/webserver/__init__.py

```diff
@@ -77,11 +78,18 @@ def gen_app(debug=None):
 
     In the Flask app returned, blueprints are not registered.
     """
+
     app = CustomFlask(
         import_name=__name__,
         use_flask_uuid=True,
     )
 
+    # As early as possible, if debug is True, set the log level of our 'listenbrainz' logger to DEBUG
+    # to prevent flask from creating a new log handler
+    if app.debug:
+        logger = logging.getLogger('listenbrainz')
+        logger.setLevel(logging.DEBUG)
+
     load_config(app)
```

Comment:
Uh! Approved too soon. Shouldn't load config happen before `app.debug` check?

Reply:
good catch, thanks. This was from some testing that I was doing and forgot to revert. It's fixed now